src/sensors/radiancemeter.cpp

```diff
@@ -77,6 +77,11 @@ MTS_VARIANT class RadianceMeter final : public Sensor<Float, Spectrum> {
 
         if (m_film->size() != ScalarPoint2i(1, 1))
             Throw("This sensor only supports films of size 1x1 Pixels!");
+
+        if (m_film->reconstruction_filter()->radius() >
+            0.5f + math::RayEpsilon<Float>)
```

Comment:
Not sure the `math::RayEpsilon` is necessary here. Am I missing someting?

I would just do the following comparison:
```
if (m_film->reconstruction_filter()->radius() > 0.5f)
```

Reply:
I initially didn't put the extra `math::RayEpsilon`, but then I was getting the warning when using the box filter with its default settings; hence this fix.

Reply:
My bad, I didn't know `math::RayEpsilon` was used in `box.cpp`. All good then!